torchtune/datasets/_samsum.py

```diff
@@ -45,11 +47,12 @@ def samsum_dataset(
         >>> Batch size: 8
     """
 
-    return InstructDataset(
+    return instruct_dataset(
         tokenizer=tokenizer,
         source=source,
-        template=SummarizeTemplate,
+        template="SummarizeTemplate",
```

Comment:
why quotes here?

Reply:
we are going through the instruct dataset builder instead of the class which requires specifying the template as a string (the builder is meant to be more config friendly)